### PR TITLE
C380546 Verify that selected filters persist on Logs tab (firebird) (…

### PR DESCRIPTION
**Pull request by KarapetyanSofia**
[C380546](https://foliotest.testrail.io/index.php?/cases/view/380546)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6479/allure)

image

### CODE REVIEW

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        ],
+        'faculty',
+      ).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.bulkEditPath,
+          waiter: BulkEditSearchPane.waitLoading,
+        });
+        FileManager.createFile(`cypress/fixtures/${userBarcodesFileName}`, user.barcode);
+        FileManager.createFile(`cypress/fixtures/${userUUIDsFileName}`, user.userId);
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js:37]: 2) User has valid .csv files with Users and Items identifiers to upload for Bulk Edit 
https://foliotest.testrail.io/index.php?/cases/view/380546

**Review by dnovikau — APPROVED** [review]

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+const editedFileName = `edited-records-${getRandomPostfix()}.csv`;
+const itemUUIDsFileName = `itemUUIDs_${getRandomPostfix()}.csv`;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+
+describe('bulk-edit', () => {
+  describe('logs', () => {
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js:20]: please just do one describe

describe('Bulk Edit - Logs', () => {

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import BulkEditSearchPane from '../../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import FileManager from '../../../../support/utils/fileManager';
+import Users from '../../../../support/fragments/users/users';
+import BulkEditActions from '../../../../support/fragments/bulk-edit/bulk-edit-actions';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+
+let user;
+const userBarcodesFileName = `userBarcodes_${getRandomPostfix()}.csv`;
+const editedFileName = `edited-records-${getRandomPostfix()}.csv`;
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js:12]: moving forward do we use this?

File: cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        const newLocation = 'Online';
+        BulkEditActions.openActions();
+        BulkEditActions.openInAppStartBulkEditFrom();
+        BulkEditActions.replaceTemporaryLocation(newLocation, 'item', 0);
+        BulkEditActions.confirmChanges();
+        BulkEditActions.commitChanges();
+        BulkEditSearchPane.verifyChangedResults('Online');
+
+        BulkEditSearchPane.openLogsSearch();
+        BulkEditSearchPane.verifyLogResultsFound();
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/logs/in-app/bulk-edit-logs-in-app-verify-selected-filters-persist-logs-tab.cy.js:99]: please verify that previously selected filters persist

**Review by nayimovag — APPROVED** [review]

**Pull request merged**